packages/cinema/cinema.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+import utils
+import tmdbsimple as tmdb
+import functools
+import random as ran
+import datetime as dt
```

Comment:
Why are you renaming :
- `tmdbsimple as tmdb`
- `random as ran`
- `datetime as dt`

We should keep the default, it's a lot clearer, and as said, I don't think abbreviations in the code is great.